osu.Game/Beatmaps/BeatmapDifficulty.cs

```diff
@@ -56,10 +56,22 @@ public static double DifficultyRange(double difficulty, double min, double mid,
         /// Maps a difficulty value [0, 10] to a two-piece linear range of values.
         /// </summary>
         /// <param name="difficulty">The difficulty value to be mapped.</param>
-        /// <param name="range">The values that define the two linear ranges.</param>
-        /// <param name="range.od0">Minimum of the resulting range which will be achieved by a difficulty value of 0.</param>
-        /// <param name="range.od5">Midpoint of the resulting range which will be achieved by a difficulty value of 5.</param>
-        /// <param name="range.od10">Maximum of the resulting range which will be achieved by a difficulty value of 10.</param>
+        /// <param name="range">The values that define the two linear ranges.
+        /// <list type="table">
+        ///   <item>
+        ///     <term>od0</term>
+        ///     <description>Minimum of the resulting range which will be achieved by a difficulty value of 0.</description>
+        ///   </item>
+        ///   <item>
+        ///     <term>od5</term>
+        ///     <description>Midpoint of the resulting range which will be achieved by a difficulty value of 5.</description>
+        ///   </item>
+        ///   <item>
+        ///     <term>od10</term>
+        ///     <description>Maximum of the resulting range which will be achieved by a difficulty value of 10.</description>
+        ///   </item>
+        /// </list>
+        /// </param>
         /// <returns>Value to which the difficulty value maps in the specified range.</returns>
         public static double DifficultyRange(double difficulty, (double od0, double od5, double od10) range)
```

Comment:
This overload is better to be removed. The only call [here](https://github.com/ppy/osu/tree/master/osu.Game/Rulesets/Scoring/HitWindows.cs#L90) passes `min, mid, max`, not `od0, od5, od10` to it.